lib/hyrax/configuration.rb

```diff
@@ -109,19 +109,41 @@ def register_roles
     attr_reader :analytics
     def analytics?
       @analytics ||=
-        ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYRAX_ANALYTICS', false))
+        ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYRAX_ANALYTICS', 'false'))
     end
 
-    attr_writer :google_analytics_id
-    def google_analytics_id
-      @google_analytics_id ||= nil
+    # Currently supports 'google' or 'matomo'
+    # google is default for backward compatability
+    attr_writer :analytics_provider
+    def analytics_provider
+      @analytics_provider ||=
+        ENV.fetch('HYRAX_ANALYTICS_PROVIDER', 'google')
+    end
+
+    # Date you wish to start collecting analytics for
+    # This is used to compute the "all-time" metrics
+    # Set this in your .env file (format is YYYY-MM-DD)
+    attr_writer :analytics_start_date
+    def analytics_start_date
+      @analytics_start_date ||=
+        ENV.fetch('ANALYTICS_START_DATE', Time.zone.today - 1.year)
     end
-    alias google_analytics_id? google_analytics_id
 
     # Defaulting analytic start date to whenever the file was uploaded by leaving it blank
     attr_writer :analytic_start_date
```

Comment:
I didn't want to remove any existing code because I was afraid it would break for people with google analytics already installed and working in their applications.  I am not totally clear on what is needed for that to work, and what is not.

Reply:
We discussed keeping the existing statistics code side by side with the new analytics code because although the new analytics code has a way better dashboard experience and fixes the views and downloads on works and filetypes, it does not do anything for the user statistics found in the user profile and their is still some unique value in the statistics dashboard option.

Since they both essential run off of the same database entries (populated by provider) they shouldn't be in conflict with one another.  I am testing this theory this week, since we have statistics working here at UC.

So it is is the recommendation of the tech call attendees to merge in the analytics PR and create a separate issue for removing the statistics, updating the user profile stats, and consolidating the dashboard views.